packages/terra-toolbar/docs/README.md

```diff
@@ -0,0 +1,23 @@
+# Terra Toolbar
```

Comment:
Can we convert this to mdx?

Reply:
Oh you have mdx. https://github.com/cerner/terra-core/pull/2875/files#diff-783660436a78f89d20bba2d9ce7a2b26R1

I think we can remove this file then